test/unit/mixins/validatable-http-message-test.js

```diff
@@ -1299,7 +1298,8 @@ Unexpected token '\\'' at 1:22
 
             describe('when a validator throws an error', () => {
               before(() => {
-                invalidSchema = require('../../fixtures/invalid-schema-v4');
+                // eslint-disable-next-line global-require
+                const invalidSchema = require('../../fixtures/invalid-schema-v4');
```

Comment:
Why not to require this at the top of the file, like

```js
const INVALID_SCHEMA = require('../../fixtures/invalid-schema-v4');
```

Reply:
To be honest, I have no idea. I think there was some intent to do it this way in the past, but I can't verify it.